.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,35 @@
+name: Bug report
+description: Report a bug to help us improve the Sail Operator
+title: "[BUG]"
+labels:
+  - "bug"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: checkboxes
+    id: security-check
+    attributes:
+      label: Is this the right place to submit this?
+      description: |-
+        This is used to report product bugs and issues that prevent you from using the operator as expected.
+        To ask questions about how to use Sail Operator, please visit <https://github.com/istio-ecosystem/sail-operator/discussions>.
+      options:
+        - label: "This is not a question about how to use the Sail Operator"
+          required: true
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Bug Description
+      description: Tell us what issues you ran into.
+      placeholder: Include information about what you tried, what you expected to happen, and what actually happened. The more details, the better!
```

Comment:
maybe also a version of the operator and possibly a link to a gist that contains relevant logs

Reply:
@dgn updated